src/ngrx/16-test-used-selectors/test-used-selectors.md

```diff
@@ -30,9 +30,29 @@ The `--watch` switch will rerun your tests whenever a code file changes. You can
 
 ## Description
 
-TODO:
+In this section will write unit tests involving Sections. Whenever we use Selectors in our Components, Services, etc. We can use `provideMockStore()` to mock the behavior of our Selectors.
```

Comment:
> “In this section we will write unit tests involving Sections..."

- @vdtdev

Reply:
@vdtdev , I've added we, but also added a comma (`,`) after the word section to match how it's punctuated in the Testing Actions page:

> In this section, we will write unit tests involving Selectors.

Reply:
> I also don’t think the “We” in the second sentence needs to be capitalized

- @vdtdev

Reply:
makes sense, done~